### PR TITLE
Update Hybrid Common 4.1.2 and add AdServices

### PR DESCRIPTION
**Pull request by joshdholtz**
Update Hybrid Common 4.1.2 and add AdServices

### CODE REVIEW

File: RevenueCat/Plugins/iOS/PurchasesUnityHelper.m [hunk 1]
```diff
@@ -139,23 +139,6 @@ - (void)syncPurchases {
     }];
 }
 
-- (void)addAttributionData:(NSString *)dataJSON network:(int)network networkUserId:(NSString * _Nullable)networkUserId {
```
Comment (aboedo) [inline RevenueCat/Plugins/iOS/PurchasesUnityHelper.m:142]: this should have been deleted in 4.0.0, my bad! It was unused, though. And it's obsoleted in C#

File: RevenueCat/Scripts/StoreProduct.cs [hunk 2]
```diff
@@ -29,9 +29,9 @@ public StoreProduct(JSONNode response)
             Identifier = response["identifier"];
             Description = response["description"];
             Price = response["price"];
-            PriceString = response["price_string"];
-            CurrencyCode = response["currency_code"];
-            var introPriceJsonNode = response["intro_price"];
+            PriceString = response["priceString"];
+            CurrencyCode = response["currencyCode"];
+            var introPriceJsonNode = response["introPrice"];
```
Comment (aboedo) [inline RevenueCat/Scripts/StoreProduct.cs:34]: support for changes in purchases-hybrid-common 4.0.0

**Comment by aboedo** [comment]
should be ready to go now!

**Review by tonidero — APPROVED** [review]
LGTM!

**Comment by aboedo** [comment]
tested on device, both Android and iOS, ready to go.

**Pull request merged**